packages/dids/src/index.js

```diff
@@ -0,0 +1,61 @@
+import {Ed25519VerificationKey2020} from '@digitalbazaar/ed25519-verification-key-2020';
+import {CryptoLD} from 'crypto-ld';
+const didKeyDriver = require('@digitalbazaar/did-method-key').driver();
+
+export const DIDKeyManager = (function () {
+  let wallet;
+
+  const createDidKeyPair = async () => {
```

Comment:
we need to use the existing polkadot keypairs and their generation (exaclty the same as used for accounts)

Reply:
this class shouldnt even care about generating keypairs at all

Reply:
> we need to use the existing polkadot keypairs and their generation (exaclty the same as used for accounts)
@SamHellawell 

Please do you have a reference to where we did this, probably in the dock certs or dock app.
Because from my understanding, we generate accounts from mnemonics in the dock app.

Reply:
the accounts are keypairs generated from a mnemonic seed. this is done in the dock app somewhere, but im not sure where. @maycon-mello can maybe help with this. essentially we just need to take a keypair document and get it to a method to format that into a did document (we need to do it this way to support did:dock later)

Reply:
we should be able to resolve the existing keypair document. Maybe could create something like `getDIDDocument(account: string)` passing the account id as arguments